private/setup-babel.js

```diff
@@ -54,6 +54,18 @@ module.exports = function setupBabel(parent) {
     if (!hasPlugin('transform-class-properties')) {
       plugins.push(requireTransform('babel-plugin-transform-class-properties'));
     }
+  } else if (checker.satisfies('^7.0.0')) {
```

Comment:
Hmm, actually can you change this to `checker.gt(‘7.0.0’)` instead?

Reply:
The reason here is that satisfies will never match a beta (not sure we will do more betas for minors and stuff, but it seems reasonable to allow it)